config/routes.php

```diff
@@ -9,10 +9,10 @@
 $app->post('/admin/users/new', 'admin.user.controller:postCreateAction')->bind('admin_users_create_post');
 $app->get('/admin/users/{userId}', 'admin.user.controller:editAction')->bind('admin_users_edit');
 
-$app->get('/admin/user/{username}/addressbooks', 'admin.user.controller:addressbookAction')->bind('admin_user_addressbooks');
-$app->get('/admin/user/{username}/calendars', 'admin.user.controller:calendarAction')->bind('admin_user_calendars');
-$app->get('/admin/user/{username}/edit', 'admin.user.controller:editAction')->bind('admin_user_edit');
-$app->get('/admin/user/{username}/delete', 'admin.user.controller:deleteAction')->bind('admin_user_delete');
-$app->post('/admin/user/{username}/delete', 'admin.user.controller:postDeleteAction')->bind('admin_user_delete_post');
+$app->get('/admin/user/{userName}/addressbooks', 'admin.user.controller:addressbookAction')->bind('admin_user_addressbooks');
```

Comment:
I think @ should be fine, as long as its urlencoded. Some DAV clients don't love it though, but i suppose that's unrelated